frontend/src/__tests__/cypress/cypress/types.ts

```diff
@@ -84,3 +84,10 @@ export type TestConfig = {
   OCP_ADMIN_USER: UserAuthConfig;
   S3: AWSS3Buckets;
 };
+
+export type DataScienceProjectData = {
+  dspCreationProjectName: string;
+  dspCreationProjectDescription: string;
+  dspCreationURL: string;
+  dspOCName: string;
```

Comment:
The types should not have the "-Creation-". Also as the type is the DSP, shouldn't contain the "dsp-" in the fields
Why it does have the OCName and the URL?

Reply:
I changed the types as requested. 

The test data in the fixtures field is pre-apprended with 'dsp..' in the 'dataScienceProject.yaml', this allows us to then build upon this test data file for other data science projects.